tests/TestCase.php

```diff
@@ -24,4 +36,20 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('auth.providers.users.model', User::class);
         $app['config']->set('passport.client_uuids', true);
     }
+
+    protected function authenticateAsAdmin(): void
+    {
+        $this->user = config('auth.providers.users.model')::factory()->create();
+        $this->user->guard_name = 'api';
+        $this->user->givePermissionTo(H5PPermissionsEnum::H5P_LIST);
```

Comment:
You are calling PermissionTableSeeder, so just ensuring that user has correct role in tests is enough, I think that these calls to givePermissionTo can be removed

Reply:
done